rouge.gemspec

```diff
@@ -15,4 +15,7 @@ Gem::Specification.new do |s|
   s.files = Dir['Gemfile', 'LICENSE', 'rouge.gemspec', 'lib/**/*.rb', 'lib/**/*.yml', 'bin/rougify', 'lib/rouge/demos/*']
   s.executables = %w(rougify)
   s.licenses = ['MIT', 'BSD-2-Clause']
+
+  s.add_development_dependency "bundler", "~> 1.10"
+  s.add_development_dependency "rake", "~> 12.0"
```

Comment:
OMG. I have misunderstood about dev dependencies.

I read http://guides.rubygems.org/patterns/ again and conclude that `Gemfile` is enough to declare dev dependencies.

Will move them to Gemfile again.